logging/src/main/java/org/wildfly/swarm/logging/LoggingFraction.java

```diff
@@ -51,6 +60,9 @@
 
     public static final String COLOR_PATTERN = "COLOR_PATTERN";
 
+    public static LoggingFraction loggingFraction;
```

Comment:
Basically, if a user starts with

    LoggingFraction logging = new LoggingFraction()

or

    LoggingFraction logging = LoggingFraction.createDebugLoggingFraction()

you should be able work with *either* of those starting points, allowing for

    logging.logToFile(...) // or whatever